src/npr/communication.py

```diff
@@ -46,39 +47,63 @@ def ship_qcreports(config, flowcell):
     samba_fdir = os.path.join(
         config['paths']['deepseq_qc'],
         'Sequence_Quality_{}'.format(yrstr),
-        'ONT_{}'.format(yrstr),
-        flowcell
+        'ONT_{}'.format(yrstr)
     )
-    # I think latency causes scp to fail so include sleep
+    # I think latency causes scp to fail so include sleep (legacy from Leily?)
     _cmd = 'mkdir -p {}'.format(samba_fdir)
     _stdin, _stdout, _stderr = client.exec_command(_cmd)
-    print('stdin:')
-    print(_stdin)
-    print('stdout:')
-    print(_stdout)
+
+#    print('stdin:')
+#    print(_stdin)
+#    print('stdout:')
+#    print(_stdout)
     print('stderr:')
     print(_stderr)
-    sleep(30)
+    sleep(10)
```

Comment:
I put this sleep in there. It's necessary as it takes a while for the mounted dir to figure out the folder was created..

Reply:
Would it be necessary to have sleep(30)? 
Please notice that there is another instance of "mkdir -p" in line 98. Do you think this requires another sleep?